lib/robots/sdr_repo/preservation_ingest/verify_apo.rb

```diff
@@ -42,9 +42,10 @@ def relationship_md_pathname
           @relationship_md_pathname if @relationship_md_pathname.file?
         end
 
-        def verify_apo_moab
-          apo_moab = Stanford::StorageServices.find_storage_object(apo_druid)
-          return if apo_moab && apo_moab.object_pathname && apo_moab.object_pathname.directory?
+        def verify_apo_registered
+          apo_object = Dor.find(apo_druid)
```

Comment:
this becomes the ONLY place in this code base where we do a Dor.find, or do anything with Dor directly.  That is another reason this approach "smells" to me.

Could we do this here for now, add the check to ?? common-accessioning ?? , and then remove it from here?   knowing that we have to address existing open workflows for WF step changes?